tests/input/datasets.yaml

```diff
@@ -0,0 +1,48 @@
+---
+"@context": https://raw.githubusercontent.com/EMMC-ASBL/oteapi-dlite/refs/heads/rdf-serialisation/oteapi_dlite/context/0.2/context.json
+
+# This extends the list of prefixes that are already defined in the context
+prefixes:
+  sem: "https://w3id.com/emmo/domain/sem/0.1#"
+  semdata: "http://sintef.no/data/matchmaker/SEM/"
+  dm: "http://onto-ns.com/meta/characterisation/0.1/SEMImage#"
+
+# List of documented datasets
+datasets:
+    # unique ID of the dataset
+  - "@id": "semdata:sample3/pos1_01_grid_200x"
+    # Dataset type as defined in the SEM domain ontology
+    "@type": "https://w3id.com/emmo/domain/sem/0.1#"
+    title: SEM image of cement
+    description: Back-scattered SEM image of cement sample 3, polished with 1 µm diamond compound.
+    creator: Sigurd Wenner
+    contactPoint: "Sigurd Wenner <Sigurd.Wenner@sintef.no>"
+
+    datamodel: http://onto-ns.com/meta/characterisation/0.1/SEMImage
+    mappingURL: https://raw.githubusercontent.com/HEU-MatCHMaker/DataDocumentation/refs/heads/master/SEM/datamodels/SEMImage.ttl
+
+    # Contextual documentation of the dataset
+    statements:
+      - ["@id", "sem:fromSample", "semdata:sample3"]
+      - ["@id", "emmo:isDescriptionOf", "semdata:concrete1"]
+
+    # A dataset can have several distributions, hence a list
+    distribution:
+      - downloadURL: https://github.com/HEU-MatCHMaker/DataDocumentation/raw/refs/heads/master/SEM/example_data/(ThermoFisher)%20pos1_01_grid_200x.tif
+        mediaType: image/tiff
+        parser:
+          parserType: application/vnd.dlite-parse
+          configuration:
+            driver: image
+            options: "plugin=tiffile"
+
+    # List of consumers of this dataset
+    datasink:
+        # This simple consumer wants the data in png format
+      - storeURL: mydata.png
+        mediaType: image/png
+        generator:
+          functionType: application/vnd.dlite-generate
+          configuration:
+            driver: image
```

Comment:
Why should a dataset have a list of consumers? Isn't one of the core ideas of this documentation framework that datasources and datasinks should be documented independently?